source/operations/install-deploy-manage/deploy-minio-single-node-single-drive.rst

```diff
@@ -17,6 +17,9 @@ Starting with :minio-release:`RELEASE.2022-06-02T02-11-04Z`, MinIO implements a
 This feature allows access to :ref:`erasure coding dependent features <minio-erasure-coding>` without the requirement of multiple drives.
 This mode **requires** accessing stored objects through the S3 API, and does **not** support direct access to objects through the filesystem/POSIX interface.
 
+Starting with :minio-release:`RELEASE.2024-01-28T22-35-53Z`, MinIO pre-allocates 1GiB of system memory at startup.
```

Comment:
This is two paragraphs in a row that start this way.
I suggest moving this down into its own "Memory Requirements" subheading to match with the other pages, like what you did on the SNMD page.